ruddock/modules/birthdays/routes.py

```diff
@@ -0,0 +1,47 @@
+import json
+import flask
+
+import datetime
+
+from ruddock.resources import Permissions
+from ruddock.decorators import login_required
+from ruddock.modules.birthdays import blueprint, helpers
+
+@blueprint.route('/')
+@login_required(Permissions.BIRTHDAYS)
+def show_bdays():
+  """Displays a list of birthdays for current students."""
+
+  # Extract parts of today's date
+  today = datetime.date.today()
+  ref_date = today.strftime("%m/%d")
+  ref_year = today.year
+
+  def process(row):
+    name = row["name"]
+    birthday = row["birthday"]
+
+    # Split the birthday into year and date
+    bdate = birthday.strftime("%m/%d")
```

Comment:
Throws `AttributeError: 'NoneType' object has no attribute 'strftime'` if user has no birthday set.
Since the database schema allows for NULL for birthday, you should degrade gracefully if the user has no birthday.